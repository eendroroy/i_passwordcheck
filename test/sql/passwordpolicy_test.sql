LOAD 'passwordpolicy';

CREATE USER test_pass WITH PASSWORD 'aaaa';

CREATE USER test_pass WITH PASSWORD 'aaaaaaaaaaaa';

CREATE USER test_pass WITH PASSWORD 'aaaaaaaa1234';

CREATE USER test_pass WITH PASSWORD 'aaaaaa#*#134';

CREATE USER test_pass WITH PASSWORD 'ASWsdf#*#134';

DROP USER IF EXISTS test_pass;