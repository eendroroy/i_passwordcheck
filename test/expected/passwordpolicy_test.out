LOAD 'passwordpolicy';
CREATE USER test_pass WITH PASSWORD 'aaaa';
ERROR:  password is too short.
CREATE USER test_pass WITH PASSWORD 'aaaaaaaaaaaa';
ERROR:  password must contain atleast 2 numeric characters.
CREATE USER test_pass WITH PASSWORD 'aaaaaaaa1234';
ERROR:  password must contain atleast 2 special characters.
CREATE USER test_pass WITH PASSWORD 'aaaaaa#*#134';
ERROR:  password must contain atleast 2 upper case letters.
CREATE USER test_pass WITH PASSWORD 'ASWsdf#*#134';
DROP USER IF EXISTS test_pass;
